resources/definitions/z-bolt_base.def.json

```diff
@@ -0,0 +1,279 @@
+			"volumic_strong_ultra",
+			"volumic_support_ultra",
+			"zyyx_pro_flex",
+			"zyyx_pro_pla",
+            "generic_abs",
+            "generic_bam",
+            "generic_cffcpe",
+            "generic_cffpa",
+            "generic_cpe",
+            "generic_cpe_plus",
+            "generic_gffcpe",
+            "generic_gffpa",
+            "generic_hips",
+            "generic_nylon",
+            "generic_pc",
+            "generic_petg",
+            "generic_pla",
+            "generic_pp",
+            "generic_pva",
+            "generic_tough_pla",
+            "generic_tpu",
+            "generic_abs_175",
+            "generic_cpe_175",
+            "generic_hips_175",
+            "generic_nylon_175",
+            "generic_pc_175",
+            "generic_petg_175",
+            "generic_pla_175",
+            "generic_pva_175",
+            "generic_tpu_175"
+        ]
+    },
+    "overrides":
+    {
+        "machine_name":                                 { "default_value": "Z-Bolt" },
+        "machine_width":                                { "default_value": 200 },
+        "machine_depth":                                { "default_value": 200 },
+        "machine_height":                               { "default_value": 200 },
+		"machine_shape":                     			{ "default_value": "rectangular" },
+        "machine_center_is_zero":            			{ "default_value": false },
+        "machine_heated_bed":                           { "default_value": true },
+        "machine_gcode_flavor":              			{ "default_value": "RepRap (Marlin/Sprinter)" },	
+        "machine_head_with_fans_polygon":               { "default_value": [ [-34, 19], [-34, -64], [34, -64], [34, 19] ] },
+        "gantry_height":                                { "value": 99999 },
+        "machine_extruder_count":            			{ "value": 1 },
+        "machine_start_gcode":                          { "default_value": "G21 ;metric values\nG90 ;absolute positioning\nM82 ;set extruder to absolute mode\nM107 ;start with the fan off\nG28 X0 Y0 Z0 ;Home all\nM140 S{material_bed_temperature_layer_0}\nM104 S160\nM190 S{material_bed_temperature_layer_0}\nM109 S160\nG4 P15000; pause\nG29;auto bed level\nG1 Z3 F1200 ;move the platform up 3mm\nG1 X0 Y0 F12000\nG1 Z0 F1200 ;move the platform down\nM104 S{material_print_temperature_layer_0}\nM109 S{material_print_temperature_layer_0}\nG1 Z3 F6000 ;move the platform up 3mm\nG92 E0;zero the extruded length\n;intro line\nG1 X0 Y0 F5000\nG1 Z0.2 F2000\nG1 X50 E10 F1500\nG1 X100 E18 F1000\nG92 E0\n" },
+        "machine_end_gcode":                            { "default_value": "G91 ;relative positioning\nG1 E-6 F1200 ;start retract\nG1 Z+3 E-10 F5000 ;continue retract\nG90 ;absolute positioning\nG28 X0 Y0\nM104 S0\nM140 S0\nM82\n" },
+
+        "layer_height_0":								{ "value": "layer_height if layer_height >= 0.24 else 0.2"},
+		"line_width":                                   { "value": "machine_nozzle_size" },
```

Comment:
Please fix the identation everywhere. It's a lot easier for the future if it's formatted the same e

Reply:
It was easier for me to develop and test profiles with this indentation, however it's not needed now since profiles are ready. Fixed it.